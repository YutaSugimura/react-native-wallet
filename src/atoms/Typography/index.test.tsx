import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Typography } from './index';

describe('renders correctly', () => {
  it('atoms/Typography (default)', () => {
    const tree = renderer.create(<Typography>text</Typography>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('atoms/Typography (props)', () => {
    const tree = renderer
      .create(
        <Typography fontColor="#1C1C1C" fontSize={12} fontWeight="700" lineHeight={16}>
          text
        </Typography>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
