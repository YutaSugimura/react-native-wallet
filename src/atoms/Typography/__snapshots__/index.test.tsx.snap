// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly atoms/Typography (default) 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "#1C1C1C",
        "fontSize": 14,
        "fontWeight": "400",
      },
      Object {
        "color": undefined,
        "fontSize": undefined,
        "fontWeight": undefined,
        "lineHeight": undefined,
      },
    ]
  }
>
  text
</Text>
`;

exports[`renders correctly atoms/Typography (props) 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "#1C1C1C",
        "fontSize": 14,
        "fontWeight": "400",
      },
      Object {
        "color": "#1C1C1C",
        "fontSize": 12,
        "fontWeight": "700",
        "lineHeight": 16,
      },
    ]
  }
>
  text
</Text>
`;
